### ISSUE TITLE
Polling without having a WebAPIKey installed makes the adapter crash

### ISSUE THREAD

**Issue by systemofapwne** [Zefau/ioBroker.nuki2#10]
When polling is enabled, a timeout preiodically calls 

https://github.com/Zefau/ioBroker.nuki2/blob/aff4d4aa8e6b2513de7f2d94eab53461f180fab4/main.js#L257

updateLocks() itself is using the WebAPI and does not check, if the nuki object is null (=no WebApiKey set) or not (WebApiKey set). In the first case, it will crash the adapter on every poll.

https://github.com/Zefau/ioBroker.nuki2/blob/aff4d4aa8e6b2513de7f2d94eab53461f180fab4/main.js#L410-L413

**Comment by Zefau** [comment]
Thanks for the info. I will fix it.

**Comment by Zefau** [comment]
Could you verify if the bug got fixed by re-installing the adapter from Github?

**Comment by systemofapwne** [comment]
It works :)

**Comment by systemofapwne** [comment]
I just had problems again:

``
nuki2.0	2019-05-07 23:01:16.175	debug	system.adapter.admin.0: logging true
nuki2.0	2019-05-07 22:57:27.676	debug	State of nuki2.0.door__XXXXXXX.action has changed {"val":0,"ack":true,"ts":1557262647667,"q":0,"from":"system.adapter.nuki2.0","user":"system.user.admin","lc":1557262647667}.
nuki2.0	2019-05-07 22:57:27.675	debug	Action applied on Bridge API.
nuki2.0	2019-05-07 22:57:27.674	info	Triggered action -UNLATCH- on Nuki XXXXXXX.
nuki2.0	2019-05-07 22:57:27.660	debug	State of nuki2.0.door__XXXXXXX.action has changed {"val":3,"ack":false,"ts":1557262647655,"q":0,"from":"system.adapter.javascript.0","user":"system.user.admin","lc":1557262647655}.
nuki2.0	2019-05-07 22:57:20.705	debug	Action applied on Bridge API.
nuki2.0	2019-05-07 22:57:20.705	info	Triggered action -UNLATCH- on Nuki XXXXXXX.
nuki2.0	2019-05-07 22:57:20.701	debug	State of nuki2.0.door__XXXXXXX.action has changed {"val":0,"ack":true,"ts":1557262640698,"q":0,"from":"system.adapter.nuki2.0","user":"system.user.admin","lc":1557262640698}.
nuki2.0	2019-05-07 22:57:20.694	debug	State of nuki2.0.door__XXXXXXX.action has changed {"val":3,"ack":false,"ts":1557262640691,"q":0,"from":"system.adapter.javascript.0","user":"system.user.admin","lc":1557262640691}.
nuki2.0	2019-05-07 22:57:04.741	debug	Action applied on Bridge API.
nuki2.0	2019-05-07 22:57:04.741	info	Triggered action -UNLATCH- on Nuki XXXXXXX.
nuki2.0	2019-05-07 22:57:04.736	debug	State of nuki2.0.door__XXXXXXX.action has changed {"val":0,"ack":true,"ts":1557262624732,"q":0,"from":"system.adapter.nuki2.0","user":"system.user.admin","lc":1557262624732}.
nuki2.0	2019-05-07 22:57:04.720	debug	State of nuki2.0.door__XXXXXXX.action has changed {"val":3,"ack":false,"ts":1557262624718,"q":0,"from":"system.adapter.javascript.0","user":"system.user.admin","lc":1557262624718}.
nuki2.0	2019-05-07 22:49:29.093	info	Successfully triggered action -LOCK- on Nuki XXXXXXX.
nuki2.0	2019-05-07 22:49:19.429	debug	Action applied on Bridge API.
nuki2.0	2019-05-07 22:49:19.428	info	Triggered action -LOCK- on Nuki XXXXXXX.
nuki2.0	2019-05-07 22:49:19.426	debug	State of nuki2.0.door__XXXXXXX.action has changed {"val":0,"ack":true,"ts":1557262159423,"q":0,"from":"system.adapter.nuki2.0","user":"system.user.admin","lc":1557262159423}.
nuki2.0	2019-05-07 22:49:19.420	debug	State of nuki2.0.door__XXXXXXX.action has changed {"val":2,"ack":false,"ts":1557262159417,"q":0,"from":"system.adapter.javascript.0","user":"system.user.admin","lc":1557262159417}.
nuki2.0	2019-05-07 22:49:08.066	debug	Callback (with URL http://XXXXXXX:/nuki-api-bridge) already attached to Nuki XXXXXXX.
nuki2.0	2019-05-07 22:49:08.064	debug	getBridgeInfo(): {"deviceType":0,"nukiId":XXXXXXX,"name":"XXXXXXX","lastKnownState":{"mode":2,"state":3,"stateName":"unlocked","batteryCritical":false,"timestamp":"2019-05-07T19:21:11+00:00"},"nuk
text2command.0	2019-05-07 22:48:51.201	info	Response: NULL
text2command.0	2019-05-07 22:48:51.194	info	Control ID scene.0.Leaving with : false
text2command.0	2019-05-07 22:48:51.193	info	processText: " tschüs[Wohnzimmer]"
``

You see, that I was able to trigger a lock event via alexa->Text2Command->Javascript->Nuki2. It triggers the action and then reports back, that the door has been locked.
When I came back, I tried to trigger an Unlatch event. Javascript->Nuki2 worked. The event has been put into "Trigger started" mode but never reports back, that it was succesful.
Calling the bridge api directly via my browsers works. Restarting the adapter (and therefore nuki-bridge-api) makes everything work again.
